tests/trinity/core/json-rpc/test_ipc.py

```diff
@@ -399,18 +405,27 @@ def uint256_to_bytes(uint):
     assert result == expected
 
 
+def mock_peer_count(count):
```

Comment:
This is an ad-hoc way to EventBus interactions. I guess we can do much better :)

Reply:
:+1: I suspect a pattern will emerge once this gets some attention.